app/commands/concept_exercise/create.rb

```diff
@@ -0,0 +1,19 @@
+class ConceptExercise
+  class Create
+    include Mandate
+
+    initialize_with :uuid, :slug, :title, :taught_concepts, :prerequisites, :deprecated, :git_sha, :synced_to_git_sha, :track
```

Comment:
I'd probably pass in the `uuid`, `track`, and then `attributes`. for the rest. You can then do `e.attributes = attributes` in the block on L9.

Reply:
(and the same comment on other places that do similar)

Reply:
```suggestion
    initialize_with :track, :uuid, :attributes
```